app/controllers/api/v1/study_files_controller.rb

```diff
@@ -335,17 +336,16 @@ def perform_update(study_file)
 
         # check if the name of the file has changed as we won't be able to tell after we saved
         name_changed = study_file.persisted? && study_file.name != safe_file_params[:name]
-
         # log the name properties to help with understanding SCP-4159
         MetricsService.log('file-update', {
-          studyAccession: @study.accession,
+          studyAccession: study.accession,
           message: 'name info',
           uploadFilename: safe_file_params[:upload_file_name],
           originalFilename: safe_file_params[:upload]&.original_filename,
           filename: safe_file_params[:name],
```

Comment:
Blocker: Mixpanel uses [fileName](https://mixpanel.com/project/2120588/view/19059/app/lexicon#transformations/event-properties/fileName), not [filename](https://mixpanel.com/project/2120588/view/19059/app/lexicon#transformations/event-properties/filename).  Consistent case will help avoid annoying analysis bugs:

```suggestion
          uploadFileName: safe_file_params[:upload_file_name],
          originalFileName: safe_file_params[:upload]&.original_filename,
          fileName: safe_file_params[:name],
```

Part of this derives from agglutinating `filename` but not e.g.  `file_type` nor `file_size`.  All other Ruby logs `fileName` instead of `filename`, and our JavaScript always uses `fileName` for this property.